edge/conf/edge.yaml

```diff
@@ -50,6 +50,10 @@ edged:
     remote-image-endpoint: /var/run/containerd/containerd.sock
     runtime-request-timeout: 2
     podsandbox-image: k8s.gcr.io/pause
+    cgroup-driver: systemd
+    node-ip: ""
```

Comment:
Why is node-ip made a flag ?

Reply:
When multiple network planes exist on the host, the user is allowed to specify the IP address of a plane as the node ip. If not specified, the loopback address is used.

Reply:
I think this flag is used to specify which IP to be considered ?
https://github.com/kubeedge/kubeedge/blob/master/edge/conf/edge.yaml#L38

Reply:
yes， but I think sometimes  there will be several IPs on the same interface maybe. I think use net interface is not a better way to get node-ip. So, when I made this piece, I used node-ip.

Reply:
Not quite sure about this. cc @qizha @kevin-wangzefeng @rohitsardesai83

Reply:
so kubelet have both, let's keep the same way with kubelet.

Reply:
OK